ledger/internal/eval.go

```diff
@@ -1566,26 +1561,62 @@ func Eval(ctx context.Context, l LedgerForEvaluator, blk bookkeeping.Block, vali
 		txvalidator.txgroups = paysetgroups
 		txvalidator.done = make(chan error, 1)
 		go txvalidator.run()
-
 	}
 
 	base := eval.state.lookupParent.(*roundCowBase)
-
 transactionGroupLoop:
 	for {
 		select {
-		case txgroup, ok := <-paysetgroupsCh:
+		case txgroup, ok := <-preloadedTxnsData:
 			if !ok {
 				break transactionGroupLoop
 			} else if txgroup.err != nil {
```

Comment:
mb log the error here with the failed group info?

Reply:
Let's improve the error string in future pr ?